cssResponsive/css-responsive_responsive-website/css/styles.css

```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
 width, etc. */
 
 /* Media Query Tablet */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
-
-/* Media Query Desktop */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
+@media screen and (min-width: 600px) {
```

Comment:
where does this value of `600px` come from?

Reply:
i used a width where i could see the changes the best (arbitrary)
600px and 900px was just so i had a better view of the changes
usually i would use 480 and 1100 for tablet and desktop